lib/extensions/ar_region.rb

```diff
@@ -12,6 +12,10 @@ module ArRegion
     cache_with_timeout(:id_to_miq_region) { Hash.new }
   end
 
+  def self.klass
+    @klass_with_ar_region ||= Class.new(ActiveRecord::Base).send(:include, self)
```

Comment:
It's not clear what this method does from the client side... ArRegion.klass.rails_sequence_start isn't obvious...

Naming is hard.

Maybe, `anonymous_class_with_ar_region` or something less verbose but clearer.